lib/ios/native-navigation/TabViewManager.swift

```diff
@@ -47,11 +47,18 @@ final class TabView: UIView {
       return viewController
     }
 
-    // TODO(lmr): handle non-RN tabs
+    let coordinator = ReactNavigationCoordinator.sharedInstance
+    
     if let route = route {
-      let vc = ReactViewController(moduleName: route, props: props).prepareViewControllerForPresenting()
-      vc.tabBarItem = tabBarItem
-      viewController = vc
+      guard let flow = coordinator.delegate?.flowCoordinatorForId?(route) as? UIViewController else {
+        let vc = ReactViewController(moduleName: route, props: props).prepareViewControllerForPresenting()
+        vc.tabBarItem = tabBarItem
+        return vc
+      }
+      (flow as! ReactFlowCoordinator).start(props)
```

Comment:
We can add other method for `ReactFlowCoordinator` in order to pass props but this is up for discussions.